src/node.cc

```diff
@@ -1584,9 +1601,9 @@ static void Cwd(const FunctionCallbackInfo<Value>& args) {
 
   size_t cwd_len = sizeof(buf);
   int err = uv_cwd(buf, &cwd_len);
-  if (err) {
+
+  if (err)
     return env->ThrowUVException(err, "uv_cwd");
-  }
```

Comment:
Can you undo the stylistic and whitespace changes?  It makes the diff a lot noisier than it needs to be.